src/compiler/utilities.ts

```diff
@@ -91,6 +91,11 @@ namespace ts {
             return false;
         }
 
+        if (sortBeforeComparison) {
+            arr1.sort();

```

Comment:
`sort` is extremely broken without a comparator (it converts each operand to a string), so I don't think it's appropriate to do this in a generic function.

Specifically, if you have a list of numbers, then `[1,2,3,10,20,30].sort() => [1, 10, 2, 20, 3, 30]`